### PR TITLE
[feat] Products listing page

### PR DESCRIPTION
**Pull request by shivampandey0**
- Homepage Navbar refactoring
- Filters for products catalog
- Products cards in the products listing

image

### CODE REVIEW

**Review by abhij1607 — APPROVED** [review]
Clean Code! Used good practices throughout

File: pages/products.html [hunk 1]
```diff
@@ -0,0 +1,339 @@
...
+          <a href="../index.html" class="nav-logo" role="img">
+            <span>MARVEL</span
+            ><span>KART<i class="fa-solid fa-cart-shopping"></i></span>
+          </a>
+        </div>
+        <div class="search">
+          <i class="fa-solid fa-magnifying-glass icon"></i>
+          <input
+            class="search-field"
+            type="text"
```
Comment (MaruthiPothuganti) [inline pages/products.html]: You can use type="search" for searchbar

**Review by MaruthiPothuganti — COMMENTED** [review]
Added one suggestion. Everything else looks good

File: pages/products.html (hunk 1 above)
Comment (shivampandey0) [reply pages/products.html]: Thanks for the suggestion.

Comment (shivampandey0) [reply pages/products.html]: Fixed.

**Pull request merged**